lang/cs/Org.Apache.REEF.Driver/Bridge/Events/AllocatedEvaluator.cs

```diff
@@ -171,5 +178,18 @@ private void ProcessNewEvaluator()
                 }
             }
         }
+
+        private IConfiguration MergeContextConfiguration(IConfiguration contextConfiguration)
```

Comment:
This could be done more efficiently by first getting all the configurations and then doing one call to `Configurations.merge()`